client/src/app.jsx

```diff
@@ -101,7 +101,7 @@ class App extends React.Component {
             QuestionSavedData ={this.state.savedQnA}
             currentItemName={this.state.currentItemName}/>
           <RatingsAndReviews
-            product_id={this.state.product_id}/>
+            product_id={28213}/>
```

Comment:
is this intentionally hard coded here?

Reply:
good call - i think it was added during presentation day for demo